package.json

```diff
@@ -42,61 +42,65 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "12.2.9",
-    "@angular/cdk": "12.2.9",
-    "@angular/common": "12.2.9",
-    "@angular/compiler": "12.2.9",
-    "@angular/core": "12.2.9",
-    "@angular/forms": "12.2.9",
-    "@angular/material": "12.2.9",
-    "@angular/platform-browser": "12.2.9",
-    "@angular/platform-browser-dynamic": "12.2.9",
-    "@angular/router": "12.2.9",
-    "@nrwl/angular": "12.9.0",
+    "@angular/animations": "13.1.1",
+    "@angular/cdk": "13.1.1",
+    "@angular/common": "13.1.1",
+    "@angular/compiler": "13.1.1",
+    "@angular/core": "13.1.1",
+    "@angular/forms": "13.1.1",
+    "@angular/material": "13.1.1",
+    "@angular/platform-browser": "13.1.1",
+    "@angular/platform-browser-dynamic": "13.1.1",
+    "@angular/router": "13.1.1",
+    "@nrwl/angular": "13.4.3",
```

Comment:
I may suggest using either `~version` or `^version` to not miss future minor/patch versions on dependencies.

Reply:
that is entirely handled by NX. I'd don't want to change a thing with these.

Reply:
I added `^` to these